### PR TITLE
ESBJAVA-4640: Orbit bundle for json schema validator by fge

### PR DESCRIPTION
**Pull request by nuwanw**

### CODE REVIEW

File: json-schema-validator-all/2.2.6.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        <version.btf>1.2</version.btf>
+        <version.jackson.coreutils>1.8</version.jackson.coreutils>
+
+        <!--OSGI version range-->
+        <version.com.google.common>[15.0.0,16.0.0)</version.com.google.common>
+        <version.javax.mail.internet>[1.4.0,1.5.0)</version.javax.mail.internet>
+        <version.org.joda.time>[1.6.0, 3.0.0)</version.org.joda.time>
+        <version.jackson.databind>[2.6.0, 3.0.0)</version.jackson.databind>
+        <version.com.fasterxml.jackson.core>[2.6.0, 3.0.0)</version.com.fasterxml.jackson.core>
+        <version.com.google.i18n.phonenumber>[6.0.0, 8.0.0)</version.com.google.i18n.phonenumber>
```
Comment (Niranjan-K) [inline json-schema-validator-all/2.2.6.wso2v1/pom.xml]: When mentioning import package range,  it is better to mention the property as "*.imp.pkg.version.range". 
please refer: https://github.com/wso2/orbit/pull/190/files

File: json-schema-validator-all/2.2.6.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        </plugins>
+    </build>
+
+    <properties>
+        <version.json.schema.validator>2.2.6</version.json.schema.validator>
+        <version.json.schema.core>1.2.5</version.json.schema.core>
+        <version.uri.template>0.9</version.uri.template>
+        <version.msg.simple>1.1</version.msg.simple>
+        <version.btf>1.2</version.btf>
+        <version.jackson.coreutils>1.8</version.jackson.coreutils>
```
Comment (Niranjan-K) [inline json-schema-validator-all/2.2.6.wso2v1/pom.xml]: As a general practice, the version is mentioned at the end of the property. Shall we follow the same approach as https://github.com/wso2/orbit/pull/190/files

File: json-schema-validator-all/2.2.6.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+ ~ limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.orbit.com.github.fge</groupId>
+    <artifactId>json-schema-validator-all</artifactId>
```
Comment (Niranjan-K) [inline json-schema-validator-all/2.2.6.wso2v1/pom.xml]: This should be the same as the artifactId that is json-schema-validator here.

Comment (nuwanw) [inline json-schema-validator-all/2.2.6.wso2v1/pom.xml]: This orbit bundle will bundle all the fge dependencies to one bundle. json-schema-validator-all will be a proper name

Comment (Niranjan-K) [inline json-schema-validator-all/2.2.6.wso2v1/pom.xml]: Noted.

**Pull request merged**